man/fi_sockets.7.md

```diff
@@ -51,6 +51,31 @@ performance numbers are lower compared to other providers implemented
 over high-speed fabric, and lower than what an application might see
 implementing to sockets directly.
 
+# RUNTIME PARAMETERS
+
+The sockets provider checks for the following environment variables. For scalability testing a user can use these environment variables to tweak the default sizes of the address vector(AV), completion queue (CQ), connection map etc. that satisfies the requriment of the particular benchmark.
+
```

Comment:
Instead, should we mention this as a side note (recommended parameters for large scale runs) ?